ubyssey/templates/about/test.html

```diff
@@ -0,0 +1,111 @@
+{% load staticfiles %}
```

Comment:
Does this file get used anywhere in production?

Reply:
I'm not too sure, I grabbed it from the advertise files when I was getting started on the template – my guess would be no because when I remove it from the test.html file everything still seems to run smoothly